packages/experimental/Button/src/Button.types.ts

```diff
@@ -122,6 +128,11 @@ export interface ButtonProps extends ButtonCoreProps {
    * @default before
    */
   iconPosition?: 'before' | 'after';
+
+  /**
+   * Button contains only icon
```

Comment:
I think it would help to be more explicit and add a statement that if this is checked, we assume there is no text content in the button.

Reply:
fixed